css/index.css

```diff
@@ -127,3 +127,93 @@ table {
   border-collapse: collapse;
   border-spacing: 0;
 }
+/* Coding Starts here */
+
+container{
+  width: 800px;
+  margin: auto;
+}
+ /* NO CSS for head!! */
+
+.body{
+  align-items: center;
+  max-width: 100px;
+  margin: 0 auto;
+  background-color: azure;
+}
+
+.nav{
+  flex-direction: row;
+  justify-content: flex-end;
+  font-size: 20px;
+
+  
+}
+.header{
+  max-width: 85%;
+  border: 1 px dashed black;
+  flex-direction: row;
+}
+
```

Comment:
Also, you'll notice once you change the `.header` CSS declaration to `header`, it will not have a 1px border around it. This is because the syntax for 1px needs to not have a space between '1' and 'px'. 

Measurements, like for borders, margins, and paddings (to name a few), need to be written as `1px`, never `1 px`.

See this MDN page for more examples and information on border:
https://developer.mozilla.org/en-US/docs/Web/CSS/border